app/src/main/res/layout/fragment_metadata.xml

```diff
@@ -1,16 +1,183 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout

```

Comment:
Why is this inside a linear layout?

Reply:
Cause it wat a linear layout before ;)

Like said before, I prefer moving over changing/improving right now. This task is big enough as it is